Screen-Sharing-Video-Chat/js/app.js

```diff
@@ -0,0 +1,169 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+var apiKey;
+var sessionId;
+var token;
+var screenPublisher;
+var screenSubscriber;
+var session;
+var cameraPublisher;
+var cameraSubscriber;
+var screenSharingButton = document.getElementById('screen-sharing');
+var elements = {'screen-subscriber': document.getElementById('screen-subscriber'), 'camera-subscriber': document.getElementById('camera-subscriber'), 'screen-publisher': document.getElementById('screen-publisher')};
+
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+function switchWithBig(pressed) {
```

Comment:
I would like to remove the `switchWithBig()`, `rearrangeElements()` and `toggleScreen()` (anything having to do with layout management). They do not strictly pertain to screen sharing, and I prefer keeping the examples terse.

Reply:
When I click a video and this function is invoked, nothing happens onscreen.

Reply:
Since you requested that we will keep the camera stream running, we are dealing with somewhere between 2 and 4 streams at all times, with the number changing dynamically.

Thus, layout management was necessary when working with the layout you commented on.

Because of that I have now changed the layout to a 2x2 "table" of streams, making the UI look not as polished but the sample more focused.

According to my testing on chrome and firefox, `switchWithBig()` did work as intended in the previous version (which means, did something only if pressing the smaller screen sharing div), yet after said layout change this isn't relevant.

Due to this change `rearrangeElements()` was removed too and I have also deleted the parts of `toggleScreen()` which deal with changing the button's css classes.